CMakeLists.txt

```diff
@@ -161,6 +161,17 @@ if(NOT APPLE AND NOT ANDROID AND NOT EMSCRIPTEN)
     set(THEXTECH_UNIX_INSTALL TRUE)
 endif()
 
+if(VITA)
+    message("Enabling PS Vita Support and fast-math flags.")
+        
+    # Fast Math flags for Vita, ensuring -DVITA is passed to the compiler.
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -ffast-math -mtune=cortex-a9 -mfpu=neon -DVITA")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ffast-math -mtune=cortex-a9 -mfpu=neon -DVITA")
+
+    # Disable annoying warning for Parameter passing.
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-psabi")
+endif()
+
```

Comment:
@Wohlstand this is what I was talking about

Reply:
By the way, such global props can be stored at "cmake/build_pros.cmake" file that I made to put the rest of global settings (and looks like, some of current set I shall move into it too, but later)